### PR TITLE
[SF7] Add Symfony 7 support for API bundle

### PR DESCRIPTION
**Pull request by loic425**
| Q               | A
|-----------------|-----
| Branch?         | symfony-7
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT

### CODE REVIEW

**Review by GSadee — APPROVED** [review]

File: src/Sylius/Bundle/ApiBundle/spec/Serializer/ProductAttributeValueNormalizerSpec.php [hunk 1]
```diff
@@ -162,12 +162,12 @@ function it_does_not_change_the_value_on_integer_type(
     ): void {
         $normalizer
             ->normalize($productAttributeValue, null, ['sylius_product_attribute_value_normalizer_already_called' => true])
-            ->willReturn(42)
+            ->willReturn([])
```
Comment (NoResponseMate) [inline src/Sylius/Bundle/ApiBundle/spec/Serializer/ProductAttributeValueNormalizerSpec.php]: Given the name of this spec, the new behaviour is invalid, or at least the test

File: src/Sylius/Bundle/ApiBundle/Serializer/Denormalizer/CommandDenormalizer.php [hunk 2]
```diff
@@ -63,6 +63,11 @@ public function denormalize($data, $type, $format = null, array $context = [])
         }
     }
 
+    public function getSupportedTypes(?string $format): array
+    {
+        return ['object' => true];
```
Comment (NoResponseMate) [inline src/Sylius/Bundle/ApiBundle/Serializer/Denormalizer/CommandDenormalizer.php:68]: We should probably play around with caching in these, but that's a topic for a later date.

Comment (GSadee) [reply src/Sylius/Bundle/ApiBundle/Serializer/Denormalizer/CommandDenormalizer.php:68]: For this denormalizer caching is enabled as you in this line

Comment (NoResponseMate) [reply src/Sylius/Bundle/ApiBundle/Serializer/Denormalizer/CommandDenormalizer.php:68]: Yeah, but the vast majority is set to `false` and I meant it in an overall sense.

**Review by NoResponseMate — APPROVED** [review]

**Comment by NoResponseMate** [comment]
Thank you, @loic425!

**Comment by NoResponseMate** [comment]
Of course, thank you @GSadee for finishing it as well 💃

**Pull request merged**